docroot/modules/custom/webform_serialization/tests/src/Functional/WebformNormalizationFormOptionsTest.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace Drupal\Tests\webform_serialization\Functional;
+
+use Drupal\Tests\BrowserTestBase;
+use Drupal\webform\Entity\Webform;
+use Drupal\Component\Serialization\Json;
+use Drupal\webform\Entity\WebformOptions;
+
+/**
+ * Tests for webform select options.
+ *
+ * @group foiaapi
+ */
+class WebformNormalizationFormOptionsTest extends BrowserTestBase {
+
+  /**
+   * Modules to enable.
+   *
+   * @var array
+   */
+  protected static $modules = [
+    'jsonapi',
+    'serialization',
+    'webform',
+    'webform_serialization',
+    'webform_template',
+    'user',
+  ];
+
+  /**
+   * Test the GET method.
+   */
+  public function testRead() {
```

Comment:
`testPopulateSelectFieldsWithOptions` to match the method we want to verify is working